src/ethereum_test_vm/opcode.py

```diff
@@ -5066,6 +5066,7 @@ class Opcodes(Opcode, Enum):
 
     EXCHANGE = Opcode(
         0xE8,
+        data_portion_length=1,
```

Comment:
late thought - one could also derive it dynamically as `len(data_portion_formatter(args))`, please advise if this is better. This should cover `RJUMPV` as well then.

Reply:
I think this should be fine as is by now :+1: